### PR TITLE
feat: add privacy manifest for iOS 17

### PR DESCRIPTION
**Pull request by Mercy811**
### Summary

Add privacy manifest file. It pretty much the same as https://github.com/amplitude/Amplitude-Swift/pull/109
- Fields are the same
- Domains are different as next-gen SDKs are using HTTP V2 

### Checklist

* [ ] Does your PR title have the correct [title format](https://github.com/amplitude/Amplitude-iOS/blob/main/CONTRIBUTING.md#pr-commit-title-conventions)?
* Does your PR have a breaking change?:

### CODE REVIEW

File: Sources/PrivacyInfo.xcprivacy [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+			<key>NSPrivacyCollectedDataTypeTracking</key>
+			<false/>
+			<key>NSPrivacyCollectedDataTypePurposes</key>
+			<array>
+				<string>NSPrivacyCollectedDataTypePurposeAnalytics</string>
+			</array>
+		</dict>
+		<dict>
+			<key>NSPrivacyCollectedDataType</key>
+			<string>NSPrivacyCollectedDataTypePreciseLocation</string>
```
[the remark this answers is not in the document]
Comment (Mercy811) [reply Sources/PrivacyInfo.xcprivacy]: You're right. Even though IP is tracked by default, it's less than 3 decimal. Changed to coarse location at https://github.com/amplitude/Amplitude-iOS/pull/476/commits/76fed56050e686f529e9822367e931d2344f7f59

**Review by justin-fiedler — APPROVED** [review]

File: Sources/PrivacyInfo.xcprivacy [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+			<false/>
+			<key>NSPrivacyCollectedDataTypePurposes</key>
+			<array>
+				<string>NSPrivacyCollectedDataTypePurposeAnalytics</string>
+			</array>
+		</dict>
+	</array>
+	<key>NSPrivacyTrackingDomains</key>
+	<array>
+		<string>https://api2.amplitude.com/</string>
```
Comment (qingzhuozhen) [inline Sources/PrivacyInfo.xcprivacy:46]: maybe worth add api.amplitude.com as well, alternative for api2.amplitude.com

Comment (Mercy811) [reply Sources/PrivacyInfo.xcprivacy:46]: I didn't see where it is used in the SDK. Could you give the link of it

Comment (qingzhuozhen) [reply Sources/PrivacyInfo.xcprivacy:46]: That's the default in JS https://github.com/amplitude/Amplitude-JavaScript/blob/main/src/constants.js#L8, and could be supported in iOS as well

Comment (Mercy811) [reply Sources/PrivacyInfo.xcprivacy:46]: This API is not in iOS SDK directly. So if customers are going to use this or other endpoints they want they have to add it by themselves. This manifest is just a reference on which endpoints can be used by the SDK by default (`https://api2.amplitude.com/`) or by some configurations changes (`https://api.eu.amplitude.com`).

Comment (Mercy811) [reply Sources/PrivacyInfo.xcprivacy:46]: Close and merge this PR but we can add it later if it's needed.

**Comment by wlxo0401** [comment]
@Mercy811 

I think this is just adding 'PrivacyInfo'.

Shouldn't it be applied to SPM and CocoaPod too??

**Pull request merged**